### PR TITLE
UX Improvements

### PR DESCRIPTION
**Pull request by Tamookk**
This PR makes multiple user experience improvements. These improvements are:

- A reason for failure is now given if the SUV2ROI process fails.
- The footer on the main page has been updated to read "@OnkoDICOM2021".
- The clinical data CSV page in the add-on options menu has had its text updated.
- "RT DOSE" in the calculate DVH prompt that appears when OnkoDICOM loads a patient has been changed to "RTDOSE".

### CODE REVIEW

**Comment by JThomas2020** [comment]
Looks good guys, just wondering, is it possible to have multiple reasons for failure? If so then you could store an an array

**Comment by Tamookk** [comment]
Due to the way SUV2ROI is done, as soon as one failure condition is satisfied the process will return. The first failure that causes this is the only one recorded for this reason. There's no need to continue to step 2 if step 1 fails, in other words, so we don't know if step 2 will fail if step 1 did!

**Review by JThomas2020 — APPROVED** [review]
Team 19, Good Job!

File: src/View/mainpage/MainPage.py [hunk 1]
```diff
@@ -220,7 +220,7 @@ def create_footer(self):
         layout_footer = QtWidgets.QHBoxLayout(self.footer)
         layout_footer.setContentsMargins(0, 0, 0, 0)
 
-        label_footer = QtWidgets.QLabel("@OnkoDICOM 2019-20")
+        label_footer = QtWidgets.QLabel("@OnkoDICOM2021")
```
Comment (tdlinh2712) [inline src/View/mainpage/MainPage.py:223]: shouldn't it be "@OnkoDICOM 2021"? I'm talking about the space between the word and number, that's it

**Review by tdlinh2712 — APPROVED** [review]
just have that one question about the space, the rest are all good to me! team 23

File: src/View/mainpage/MainPage.py (hunk 1 above)
Comment (Tamookk) [reply src/View/mainpage/MainPage.py:223]: As it is in the PR, with no space, is exactly how the product owner requested it in Slack. This was also shown off during our recent meeting and no complaints were raised!

Comment (tdlinh2712) [reply src/View/mainpage/MainPage.py:223]: all good then!

**Pull request merged**